source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1558,8 +1555,8 @@ the client MUST clear its connection pool to the server.
 error messages to detect "not master" and "node is recovering"`_, and `other
 transient errors`_ and `Why close connections when a node is shutting down?`_.)
 
-Monitoring SDAM events
-''''''''''''''''''''''
+SDAM events
```

Comment:
What motivated this change?

Reply:
This change is no longer part of this PR.